.github/workflows/pr-build.yml

```diff
@@ -26,4 +26,16 @@ jobs:
       - name: Build Docker container
         shell: sh
         run: |
-          docker build --pull --no-cache --force-rm .
+          docker build --pull --no-cache --tag built-manual-image .
+          docker create --name built-manual-container built-manual-image
+          docker cp built-manual-container:/usr/local/apache2/htdocs "$RUNNER_TEMP/htdocs"
+          docker container rm --force --volumes built-manual-container
```

Comment:
I think you can avoid keeping the temporary container across multiple commands by replacing this with something like:

```
docker run --rm built-manual-image tar -C /usr/local/apache2 -c htdocs | tar -C "$RUNNER_TEMP" -x
```